ws/static/template/editable-signup-list.html

```diff
@@ -40,7 +40,7 @@
         <div data-ng-if="signup.feedback.length && !hideFeedback">
           <label><i class="fas fa-comment"></i>&nbsp;Feedback</label>
           <ul class="list-unstyled" data-ng-repeat="feedback in signup.feedback">
-            <li>{{ feedback.comments }} - <small>{{ feedback.leader }} on <a data-ng-href="/trips/{{feedback.trip.id}}/" data-ng-bind="feedback.trip.name"></a></small></li>
+            <li class="signup-list-feedback">{{ feedback.comments }} - <small>{{ feedback.leader }} on <a data-ng-href="/trips/{{feedback.trip.id}}/" data-ng-bind="feedback.trip.name"></a></small></li>
```

Comment:
Hm, adding just a newline as-is makes the signups a lot harder to visually break apart from one another. Previously, when all whitespace was treated as standard HTML whitespace, we had a nice visual "block" for each bit of feedback. Honoring newlines makes it seem like they have multiple bits of feedback:

<img width="1154" alt="unclear" src="https://user-images.githubusercontent.com/901169/51157967-1c628f80-1837-11e9-8280-d5fcfd60ea21.png">

Also, this means that leaders who accidentally add a couple newlines could really make the whole view look quite ... odd. (Leaders sometimes do leave a bunch of extra newlines):

<img width="1163" alt="screen shot 2019-01-14 at 20 01 47" src="https://user-images.githubusercontent.com/901169/51158013-474ce380-1837-11e9-988c-7727496f38dd.png">